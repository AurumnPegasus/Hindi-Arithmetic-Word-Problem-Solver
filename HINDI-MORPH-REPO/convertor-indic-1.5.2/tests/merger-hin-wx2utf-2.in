<Sentence id='2'>
1	((	NP	
1.1	mere	PRP	<fs af='mEM,pn,any,sg,1,o,kA,kA' agr_num="pl" agr_cas="o" agr_gen="m"  >
1.2	pAsa	NST	<fs af='pAsa,adv,,,,,,'  >
	))
2	((	NP	
2.1	((	NP	<fs etype="numx" value="2300.0"  >
2.1.1	((	NP	<fs etype="numx" value="2000.0"  >
2.1.1.1	2	QCC	<fs af='2,num,,,,,,' etype="numx" value="2.0"  >
2.1.1.2	hajAra	QC	<fs af='hajAra,adj,any,pl,,d,,' etype="numx" value="1000.0"  >
	))
2.1.2	((	NP	<fs etype="numx" value="300.0"  >
2.1.2.1	3	QCC	<fs af='3,num,,,,,,' etype="numx" value="3.0"  >
2.1.2.2	sO	QC	<fs af='sO,adj,any,pl,,d,,' etype="numx" value="100.0"  >
	))
	))
2.2	kiwAbeM	NN	<fs af='kiwAba,n,f,pl,3,d,0,0'  >
	))
3	((	VGF	
3.1	hEM	VM	<fs af='hE,v,any,pl,3,,hE,hE'  >
3.2	.	SYM	<fs af='.,punc,,,,,,'  >
	))
</Sentence>

