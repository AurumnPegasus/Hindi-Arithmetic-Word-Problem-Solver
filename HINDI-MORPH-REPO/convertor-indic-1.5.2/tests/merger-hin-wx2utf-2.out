<Sentence id='2'>			
1	((	NP	
1.1	मेरे	PRP	<fs af='मैं,pn,any,sg,1,o,का,kA' agr_gen="m"  agr_cas="o"  agr_num="pl">
1.2	पास	NST	<fs af='पास,adv,,,,,,'>
	))
2	((	NP	
2.1	((	NP	<fs  etype="numx"  value="2300.0">
2.1.1	((	NP	<fs  etype="numx"  value="2000.0">
2.1.1.1	2	QCC	<fs af='2,num,,,,,,' etype="numx"  value="2.0">
2.1.1.2	हजार	QC	<fs af='हजार,adj,any,pl,,d,,' etype="numx"  value="1000.0">
	))
2.1.2	((	NP	<fs  etype="numx"  value="300.0">
2.1.2.1	3	QCC	<fs af='3,num,,,,,,' etype="numx"  value="3.0">
2.1.2.2	सौ	QC	<fs af='सौ,adj,any,pl,,d,,' etype="numx"  value="100.0">
	))
	))
2.2	किताबें	NN	<fs af='किताब,n,f,pl,3,d,0,0'>
	))
3	((	VGF	
3.1	हैं	VM	<fs af='है,v,any,pl,3,,है,hE'>
3.2	.	SYM	<fs af='.,punc,,,,,,'>
	))
</Sentence>			
			
